ops/pebble.py

```diff
@@ -1316,7 +1316,7 @@ class NoticeType(enum.Enum):
     CUSTOM = 'custom'
 
 
-class NoticesSelect(enum.Enum):
+class NoticesUsers(enum.Enum):
```

Comment:
On the next line, let's fix the docstring by changing ``select`` to ``users``.

While we're there, how about we also avoid the word "select" in the ALL docstring below. "Select notices..." -> "Return notices..."